.pre-commit-config.yaml

```diff
@@ -19,29 +19,42 @@ repos:
     -   id: double-quote-string-fixer
     -   id: end-of-file-fixer
     -   id: file-contents-sorter
-        files: .gitignore
+        files: .(docker|git)ignore
     -   id: mixed-line-ending
         args: [--fix=lf]
     -   id: name-tests-test
     -   id: requirements-txt-fixer
     -   id: trailing-whitespace
--   repo: git://github.com/pre-commit/mirrors-pylint
-    sha: v1.8.4
+-   repo: https://github.com/pre-commit/mirrors-pylint
+    rev: v2.3.1
     hooks:
     -   id: pylint
         args:
+        - --max-line-length=80
+        - --ignore-imports=yes
         - -d duplicate-code
         - -d fixme
         - -d import-error
         - -d invalid-name
         - -d locally-disabled
         - -d missing-docstring
--   repo: git://github.com/Lucas-C/pre-commit-hooks
-    sha: v1.1.5
+        - -d too-few-public-methods
+        - -d too-many-arguments
+        - -d useless-object-inheritance # Necessary for Python 2 compatibility
```

Comment:
This flags things like `class Token(object):` and wants to write `class Token:` instead. But those two things are different in Python 2.